nilearn/connectome/connectivity_matrices.py

```diff
@@ -397,13 +397,13 @@ class ConnectivityMeasure(BaseEstimator, TransformerMixin):
 
     kind : {"correlation", "partial correlation", "tangent",\
             "covariance", "precision"}, optional
```

Comment:
You might consider this out-of-scope, which is totally understandable, I think convention is to place the default first in a set of options. I'm not sure how well that convention is followed in `nilearn`'s docstrings though.